New pattern to update module

vlopezferrando opened the pull request.

morgante reviewed (approved):
Looks fine. Ideally we would include/add a test case.

vlopezferrando commented:
I have added a test. I don't think there's a standard way to test multi-file patterns, so I added a directory with some fixtures with input and expected output and a script to run it.
